config.lib

```diff
@@ -2875,11 +2875,19 @@ detect_fontconfig() {
 }
 
 detect_icu_layout() {
-	detect_pkg_config "$with_icu_layout" "icu-lx" "icu_layout_config" "4.8" "1"
+	if [ "$os" = "OSX" ] && [ "$with_icu_layout" = "1" ]; then
```

Comment:
would it be better if this check was in the level above, like it is for uniscribe?

Reply:
My intention was to have the output log line in the same place with and without ICU, but YMMV here.

Reply:
Also, skipping libfontconfig is handled exactly the same (and just above in the file).